helm/README.md

```diff
@@ -0,0 +1,217 @@
+# Prometheus
+
+Here it is a new opinionated [Prometheus](https://prometheus.io/) recipe thought to be deployed in the Giant Swarm tenant clusters.
+
+## Prerequisites
+
+- Kubernetes 1.3+ with Beta APIs enabled
```

Comment:
Beta APIs?

Reply:
Copied from the original stable chart. I guess at that time you needed beta api gates